.storybook/preview.js

```diff
@@ -1 +1 @@
-import "./story.css";
+import "!style-loader!css-loader!sass-loader!../node_modules/@mozilla-protocol/core/protocol/css/protocol.scss";
```

Comment:
My current blocker: I had to use sass-loader so storybook can read scss, but the snapshot test doesn't like this. Trying to figure out a way to work around this